lib/lint-messages.js

```diff
@@ -0,0 +1,66 @@
+/** Lint messages, find missing messages and those that are unused */
+
+const YAML = require('yamljs')
+const { readFileSync } = require('fs')
+const glob = require('glob')
+
+const IMPORT = /import ([a-zA-Z0-9]+) from '[^ ]+\/messages'/
+
+module.exports = function () {
+  // TODO don't hardwire paths
+  const messages = YAML.parse(readFileSync('configurations/default/messages.yml', 'utf8'))
+
+  const missing = [].concat(...glob
+    .sync('lib/**/*.js')
```

Comment:
this should be configurable.  The root directory may not always be named this.  Also, sometimes there may be multiple root directories.